amm/contracts/pair/Cargo.toml

```diff
@@ -14,7 +14,7 @@ scale-info = { version = "2.9", default-features = false, features = [
     "derive",
 ], optional = true }
 
-psp22 = { version = "0.2" , default-features = false }
+psp22 = { version = "0.2.2" , default-features = false }
```

Comment:
This should be `=0.2.2` so that it doesn't upgrade to `0.2.X`.

Reply:
Fixed to `=` everywhere